pkg/cache/cache_test.go

```diff
@@ -96,6 +96,14 @@ func CacheTest(createCacheFunc func(config *rest.Config, opts cache.Options) (ca
 			Expect(cfg).NotTo(BeNil())
 
 			By("creating three pods")
+			cl, err := client.New(cfg, client.Options{})
```

Comment:
this is because of https://github.com/kubernetes-sigs/controller-runtime/pull/787#issuecomment-581039898